tests/dm_test.py

```diff
@@ -39,6 +39,34 @@ def _clean_up(self):
 
         os.unlink(self.dev_file)
 
+class DevMapperGetSubsystemFromName(DevMapperTestCase):
+    def _destroy_lvm(self):
+        run("vgremove --yes libbd_dm_tests >/dev/null 2>&1")
+        run("pvremove --yes %s >/dev/null 2>&1" % self.loop_dev)
+
+    def _destroy_crypt(self):
+        run("cryptsetup close libbd_dm_tests-subsystem_crypt")
+                #>/dev/null 2>&1")
```

Comment:
This looks like something you forgot to delete :)

Reply:
Indeed I did. Fixed.